src/CommunityToolkit.Aspire.OllamaSharp/OllamaSharpSettings.cs

```diff
@@ -8,7 +8,7 @@ public sealed class OllamaSharpSettings
     /// <summary>
     /// Gets or sets the connection string.
     /// </summary>
-    public string? ConnectionString { get; set; }
```

Comment:
No, I'd prefer to go with a hard-break in v9 because we have a very significan overhaul of the Ollama and OllamaSharp integrations coming that I almost consider the v8 versions completely separate.